tests/integration/courses.test.js

```diff
@@ -4134,6 +4144,44 @@ describe('COURSES ROUTES - DELETE /courses/{_id}/companies{companyId}', () => {
     });
   });
 
+  describe('HOLDING_ADMIN', () => {
+    it('should return a 200 if holding admin try to remove company from intra_holding course', async () => {
```

Comment:
idem try to

Reply:
tu peux simplement mettre `should remove company from intra_holding course`

Reply:
done